tools/update-locked-requirements

```diff
@@ -10,16 +10,47 @@ fi
 
 source /srv/zulip-venv/bin/activate # Activate python2 virtualenv
 pip-compile --output-file requirements/py2_common_lock.txt requirements/py2_common.txt
+pip-compile --output-file requirements/py2_dev_lock.txt requirements/dev.txt
 # Remove the editable flag in the lock file for safer build
 sed -i 's/-e //' requirements/py2_common_lock.txt
+sed -i 's/-e //' requirements/py2_dev_lock.txt
+# Remove duplicate dependencies manually
+# This means we have the info of which dependencies are duplicated
+sed -i '/boto==/d' requirements/py2_dev_lock.txt
+sed -i '/click==/d' requirements/py2_dev_lock.txt
+sed -i '/cryptography==/d' requirements/py2_dev_lock.txt
+sed -i '/lxml==/d' requirements/py2_dev_lock.txt
+sed -i '/pbr==/d' requirements/py2_dev_lock.txt
+sed -i '/pyasn1-modules==/d' requirements/py2_dev_lock.txt
+sed -i '/requests==/d' requirements/py2_dev_lock.txt
+sed -i '/werkzeug==/d' requirements/py2_dev_lock.txt
+# Python 2-only duplicate dependencies
+sed -i '/certifi==/d' requirements/py2_dev_lock.txt
+sed -i '/pyasn1==/d' requirements/py2_dev_lock.txt
+sed -i '/pycparser==/d' requirements/py2_dev_lock.txt
+sed -i '/pyopenssl==/d' requirements/py2_dev_lock.txt
```

Comment:
Hrm, I would have preferred we just do this in an order that works with both Python 2 and 3, so we could just merge this.  But I suppose we can wait for #6062; shouldn't be too long now.